Formula/l/lexido.rb

```diff
@@ -0,0 +1,22 @@
+class Lexido < Formula
+  desc "Innovative assistant for the command-line"
+  homepage "https://github.com/micr0-dev/lexido"
+  url "https://github.com/micr0-dev/lexido/archive/refs/tags/v1.3.1.tar.gz"
+  sha256 "8d05016b392fa43a33d6989c1f6e568f0d8b5d8893faf8622d02d36fd2d5e9db"
+  license "AGPL-3.0-or-later"
+
+  depends_on "go" => :build
+  depends_on "ollama" => :test
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+  end
+
+  test do
+    # Set a fake API key for testing purposes
+    ENV["OLLAMA_LOCAL"] = "true"
```

Comment:
As the creator of lexido, I can confirm. Also this isn't even an environment variable that lexido uses or looks at. The only one is the Google API key which is a legacy method of getting the API key.

Reply:
In the keyring file there's two under the Google api one so I set one to true because on my pc I had to set it true for some reason if that isn't right I will change that on the next push

Reply:
The keyring and environment variables are different. Things in the keyring are only read in the keyring. All you have to do is pass the `-l` flag to get it to run locally

Reply:
Alright I'm not sure why my setup made me put the in there could be because I'm running on a docker container atm till I get a new laptop but I will change that